Src/AutoFixture/MapCreateManyToEnumerable.cs

```diff
@@ -25,6 +25,12 @@ namespace AutoFixture
     /// </para>
     /// </remarks>
     /// <seealso cref="MultipleToEnumerableRelay" />
+    [Obsolete("This relay is obsolete and will be removed in future versions. " +
+              "The reason is that it causes recursion overflow for many request if you forget to add IEnumerable<T> customization. " +
+              "Instead, please use the following code snippet:\r\n" +
+              "fixture.Customizations.Add(new FilteringSpecimenBuilder(new FixedBuilder(SEQUENCE_TO_RETURN), " +
+              "new EqualRequestSpecification(new MultipleRequest(new SeededRequest(typeof(T), default(T)))))).\r\n" +
+              "If you need that often, please create an extension method inside your project.")]
```

Comment:
It'd be nice if you format those code snippets to be <= 80 chars. Most output windows (and terminals) use that width by default and most chances are that the error message won't look pretty:

```c#
fixture.Customizations.Add(
    new FilteringSpecimenBuilder(
        new FixedBuilder(
            SEQUENCE_TO_RETURN),
    new EqualRequestSpecification(
        new MultipleRequest(
            new SeededRequest(
                typeof(T),
                default(T))))))
```

Reply:
Fixed!